[2021-03-07 19:20:48,239] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_from_clickhouse_2.save 2021-03-06T00:00:00+00:00 [queued]>
[2021-03-07 19:20:48,253] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_from_clickhouse_2.save 2021-03-06T00:00:00+00:00 [queued]>
[2021-03-07 19:20:48,253] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 19:20:48,253] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-07 19:20:48,253] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 19:20:48,261] {taskinstance.py:1063} INFO - Executing <Task(_PythonDecoratedOperator): save> on 2021-03-06T00:00:00+00:00
[2021-03-07 19:20:48,266] {standard_task_runner.py:52} INFO - Started process 47 to run task
[2021-03-07 19:20:48,270] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_from_clickhouse_2', 'save', '2021-03-06T00:00:00+00:00', '--job-id', '153', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/get_from_clickhouse_2/main.py', '--cfg-path', '/tmp/tmpya_uzgyd', '--error-file', '/tmp/tmp3xs6hz2b']
[2021-03-07 19:20:48,270] {standard_task_runner.py:77} INFO - Job 153: Subtask save
[2021-03-07 19:20:48,304] {logging_mixin.py:104} INFO - Running <TaskInstance: get_from_clickhouse_2.save 2021-03-06T00:00:00+00:00 [running]> on host 020f66ce59e6
[2021-03-07 19:20:48,337] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=iotoi
AIRFLOW_CTX_DAG_ID=get_from_clickhouse_2
AIRFLOW_CTX_TASK_ID=save
AIRFLOW_CTX_EXECUTION_DATE=2021-03-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-06T00:00:00+00:00
[2021-03-07 19:20:48,403] {logging_mixin.py:104} INFO - [(0, 'somename', '192.168.0.1', '3892432'), (0, 'somename2', '192.168.0.1', '3892432'), (0, 'somename2', '192.168.0.1', '3892432'), (0, 'somename3', '192.168.0.1', '3892432'), (0, 'somename3', '192.168.0.1', '3892432'), (0, 'somename3', '192.168.0.1', '3892432'), (0, 'somename2', '192.168.0.1', '3892432')]
[2021-03-07 19:20:48,425] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=get_from_clickhouse_2, task_id=save, execution_date=20210306T000000, start_date=20210307T192048, end_date=20210307T192048
[2021-03-07 19:20:48,482] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-07 19:20:48,522] {local_task_job.py:146} INFO - Task exited with return code 0
